guides/common/modules/proc_rex-pull-based-transport.adoc

```diff
@@ -1,7 +1,7 @@
 [id="REX_pull_based_transport_{context}"]
 = REX pull-based transport
 
-{Project} has a pull-based transport that has the features of REX but with a deployment model similar to katello-agent. 
+{Project} has a pull-based transport that has the features of REX with a deployment model.
```

Comment:
Drop this line all together

Reply:
That will also mean reworking the next sentence, since it refers to "This transport"

Reply:
@jeremylenz I will be pushing another commit to have the next sentence reworded as:
**{Project} uses MQTT as its messaging protocol and includes an MQTT client running on each host.
In order to tune the MQTT server (`mosquitto`) and increase the number of hosts connected to it, you can follow the following procedure on the {ProjectServer}:**